vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_fields.html.haml

```diff
@@ -26,3 +26,6 @@
   .row
     .span8
       = f.submit text("shared.search"), class: "btn"
+      - if params[:return_path].present?
+        = link_to t("shared.cancel"), params[:return_path]
+        = hidden_field_tag "return_path", params[:return_path]
```

Comment:
We should probably do some sort of sanitization on this parameter. I can modify this in the URL to make me go anywhere I want.